lib/android_safetynet/attestation_response.rb

```diff
@@ -91,8 +89,10 @@ def trust_store
       self.class.trust_store
     end
 
-    def leaf_certificate
-      certificate_chain[0]
+    def certificate_chain
+      @certificate_chain ||= headers[CERTIRICATE_CHAIN_HEADER].map do |cert|
+        OpenSSL::X509::Certificate.new(Base64.strict_decode64(cert))
+      end
```

Comment:
Is this just a reordering?

Reply:
This is making `#leaf_certificate` a public method and `#certificate_chain` a private one, because from the outside we changed what we call.